packages/main/src/components/user-name/UserName.jsx

```diff
@@ -1,12 +1,56 @@
-import React from "react";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
 import styles from "../user-name/SettingsTab.module.css";
 
 const UserName = () => {
+  const [userName, setUserName] = useState("");
+  const [user, setUser] = useState(null);
+
+  let organization_id = JSON.parse(sessionStorage.organisations)[0].id;
+  let user_member_id = JSON.parse(sessionStorage.organisations)[0].member_id;
+
+  useEffect(() => {
+    const user = JSON.parse(sessionStorage.getItem("user"));
+    if (user) {
+      setUser(user);
+    }
+  }, []);
+
+  const updateUserName = event => {
+    event.preventDefault();
+    axios
+      .patch(
+        `https://api.zuri.chat/organizations/${organization_id}/members/${user_member_id}/profile`,
```

Comment:
use the base url constant provided in the app
it should be coming from `@zuri-utilities`